src/nvim/ex_docmd.c

```diff
@@ -3275,7 +3275,7 @@ void ex_ni(exarg_T *eap)
 {
   if (!eap->skip)
     eap->errmsg = (char_u *)N_(
-        "E319: Sorry, the command is not available in this version");
+        "E319: The command is not available in this version");
```

Comment:
Actually, I'm not so sure about these changes. Are you aware of the work it creates for the messages maintainers? I'm not saying that these messages shouldn't be improved but please realise that a minor word change results in all i18n going out of sync, probably for many years to come.